dev/benchmarks/complex_layout/windows/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(BINARY_NAME "complex_layout")
 
 # Explicitly opt in to modern CMake behaviors to avoid warnings with recent
 # versions of CMake.
-cmake_policy(SET CMP0063 NEW)
+cmake_policy(VERSION 2.4...3.25)
```

Comment:
The minimum required version (line 6) is 3.14, so you can change `2.4` to `3.14`.